app/assets/stylesheets/provider/_tables.scss

```diff
@@ -99,6 +99,9 @@ table.data {
     td.actions {
       text-align: right;
     }
+    td b.metric-name {
```

Comment:
Is the `b` necessary?

Reply:
No, but be sure to check if we have no other tables with this class. The more generic the rule, the higher the chance it will leak through. But agreed, in general style rules should not use elements at all, they should be 100% class based.

Reply:
Right now, it is. 
We could change the view, but I think it'd be another task

Reply:
`.metric-name` class is being used in other context

Not sure if break-word is what we want on all fields. Prefer doing small changes. Too many tables to check.

https://github.com/3scale/porta/blob/master/app/assets/stylesheets/provider/_stats.scss#L31
https://github.com/3scale/porta/blob/master/app/assets/stylesheets/provider/_charts.scss#L16

Reply:
@josemigallas `why not applying the break-word to all fields in the table?`
I tried, but with very large words, ends like this:
![all-break](https://user-images.githubusercontent.com/13486237/48274358-6b5d0480-e443-11e8-86ea-fabee6e18334.png)

Reply:
> I tried, but with very large words, ends like this:

😱 

> maybe a better classname would be .u-break-word

+1